src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.ts

```diff
@@ -348,6 +349,55 @@ export class RgwMultisiteDetailsComponent implements OnDestroy, OnInit {
     return this.showMigrateAction;
   }
 
+  isDeletedDisabled(node: TreeNode): boolean {
+    let disable: boolean = false;
+    let nMasterZonegroup: number = 0;
```

Comment:
```suggestion
    let masterZonegroupCount: number = 0;
```
Bad naming convention here.